rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/providers/util/LocalJobFriendlyPackerCommandFactory.groovy

```diff
@@ -32,6 +32,7 @@ class LocalJobFriendlyPackerCommandFactory implements PackerCommandFactory {
 
     packerCommand << absoluteTemplateFilePath
     packerCommand.removeAll([null])
+    packerCommand.removeAll("")
```

Comment:
Ok, then this should probably be just one line:
`packerCommand.removeAll([null, ""])`